src/js/components/AppPageComponent.jsx

```diff
@@ -343,11 +343,8 @@ var AppPageComponent = React.createClass({
     var volumes = AppsStore.getVolumes(model.id);
 
     var tabs = state.tabs.filter(tab =>
-      tab.text !== "Volumes" || (
-        model.container != null &&
-        model.container.volumes != null &&
-        model.container.volumes.length > 0
-      ));
```

Comment:
Good point – in fact, what I actually meant was: 

Do you agree that we don't want to show the Volumes tab unless ~~at least a task is running~~ **there is at least one task in any status**?

@aldipower it seems like we can proceed with the merge unless you think otherwise! Thanks